analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd

```diff
@@ -75,16 +75,28 @@ tel_stranded_file <- file.path(telomerase_dir,
 #### Import and process the metadata
 
 ```{r Read in metadata}
+# Will remove the below patients because I do not know if they had total or partial resections until we check with Jenn Mason
+
+# 7316-116 has Biopsy only;Gross/Near total resection
+# 7316-182 has Gross/Near total resection;Partial resection
+
 # get primary tumor and remove cell lines
 histologies_rna <- read_tsv(metadata_file, guess_max = 10000) %>%
   filter(composition=="Solid Tissue",
                   tumor_descriptor == "Initial CNS Tumor",
                   experimental_strategy == "RNA-Seq", 
                   RNA_library == "stranded") %>%
   rename(Kids_First_Biospecimen_ID_RNA = Kids_First_Biospecimen_ID) %>% 
-  select(Kids_First_Participant_ID, Kids_First_Biospecimen_ID_RNA, cancer_group, OS_status, OS_days, PFS_days) %>%
+  # clean up extent of tumor resection
+  mutate(extent_of_tumor_resection = case_when(
+    extent_of_tumor_resection == "Biopsy only;Gross/Near total resection" ~ "Gross/Near total resection",
+    extent_of_tumor_resection == "Gross/Near total resection;Partial resection" ~ "Gross/Near total resection",
```

Comment:
Depending on previous comment about how these groups are defined, might be able to something simpler - 

```suggestion
    str_detect(extent_of_tumor_resection, "resection") ~ "Gross/Near total resection",
```